app/controllers/sections/consignee/ConsigneeExemptOrganisationController.scala

```diff
@@ -0,0 +1,74 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.sections.consignee
+
+import controllers.BaseNavigationController
+import controllers.actions._
+import forms.sections.consignee.ConsigneeExemptOrganisationFormProvider
+import models.Mode
+import models.requests.DataRequest
+import navigation.Navigator
+import pages.sections.consignee.ConsigneeExemptOrganisationPage
+import play.api.data.Form
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import services.{GetMemberStatesService, UserAnswersService}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.select.SelectItem
+import views.html.sections.consignee.ConsigneeExemptOrganisationView
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class ConsigneeExemptOrganisationController @Inject()(override val messagesApi: MessagesApi,
+                                                      override val userAnswersService: UserAnswersService,
+                                                      override val navigator: Navigator,
+                                                      override val auth: AuthAction,
+                                                      override val getData: DataRetrievalAction,
+                                                      override val requireData: DataRequiredAction,
+                                                      override val userAllowList: UserAllowListAction,
+                                                      formProvider: ConsigneeExemptOrganisationFormProvider,
+                                                      val controllerComponents: MessagesControllerComponents,
+                                                      view: ConsigneeExemptOrganisationView,
+                                                      getMemberStatesService: GetMemberStatesService
+                                                     ) extends BaseNavigationController with AuthActionHelper {
+
+  def onPageLoad(ern: String, lrn: String, mode: Mode): Action[AnyContent] =
+    authorisedDataRequestAsync(ern, lrn) { implicit request =>
+      renderView(Ok, fillForm(ConsigneeExemptOrganisationPage, formProvider()), mode)
+    }
+
+  def onSubmit(ern: String, lrn: String, mode: Mode): Action[AnyContent] =
+    authorisedDataRequestAsync(ern, lrn) { implicit request =>
+      formProvider().bindFromRequest().fold(
+        formWithErrors =>
+          renderView(BadRequest, formWithErrors, mode),
```

Comment:
You could collapse this to _(but might be less readable)_ :
```suggestion
        renderView(BadRequest, _, mode),
```